components/org.wso2.carbon.identity.provider/src/main/java/org/wso2/carbon/identity/provider/openid/util/OpenIDUtil.java

```diff
@@ -213,23 +213,23 @@ public static String getRelyingPartyUrl(String rpUrl) throws IdentityException {
     }
 
     public static List<OpenIDParameterDTO> getOpenIDAuthRequestAsList(HttpServletRequest request) {
-        Map parameterMap = null;
-        Iterator keysIter = null;
+        Map<Object, Object> parameterMap = null;
         List<OpenIDParameterDTO> params = null;
         OpenIDParameterDTO param = null;
 
         parameterMap = request.getParameterMap();
-        keysIter = parameterMap.keySet().iterator();
         params = new ArrayList<OpenIDParameterDTO>();
```

Comment:
Can we get variable creation and initialization of parameterMap and params to one place

Reply:
fixed with https://github.com/wso2-extensions/identity-inbound-auth-openid/pull/45